docs/developers_guide/copyright_license.md

```diff
@@ -145,15 +145,15 @@ Adding copyright notices is appropriate only for __substantial__ modifications.
 ## Frequently asked questions
 
 __Can RST be released with a different license?__<br/>
-To release RST with a different license, we would need permission from all of the copyright holders. This means that, in practice, it would be very difficult to re-license the software. This is by design of the GPL license--it ensures that the software can always be used freely. 
+To release RST with a different license, we would need permission from all of the copyright holders. This means that, in practice, it would be very difficult to re-license the software. This is by the design of the GPL license--it ensures that the software can always be used freely. 
 
 
 __Can JHU/APL revoke the GPL license and take back control of RST?__<br/>
-No. JHU/APL gave permission for the RST to be re-licensed under the GPL in ~2012 (RST3.5), and they cannot revoke this permission. JHU/APL still owns the copyright to their own code, and RST versions 3.4 and earlier are still licensed by JHU/APL (not under GPL).
+No. JHU/APL permitted for the RST to be re-licensed under the GPL in ~2012 (RST3.5), and they cannot revoke this permission. JHU/APL still owns the copyright to their code, and RST versions 3.4 and earlier are still licensed by JHU/APL (not under GPL).
```

Comment:
You can't say "permitted for" -- it would be just "permitted". But I prefer it the way it was

```suggestion
No. JHU/APL gave permission for the RST to be re-licensed under the GPL in ~2012 (RST3.5), and they cannot revoke this permission. JHU/APL still owns the copyright to their code, and RST versions 3.4 and earlier are still licensed by JHU/APL (not under GPL).
```